assets/map-layers.config.json

```diff
@@ -2768,6 +2768,63 @@
       "date": null
     }
   },
+  {
+    "id": "rtsgs",
+    "layers": ["gevaarlijke_stoffen"],
+    "legendItems": [
```

Comment:
The legendItems field can be deleted, as these layers have just one legenditem. The title must then be added as imageRule to get the proper legenditem icon from the mapserver

Reply:
Should be fixed now